blockchain/genesis.go

```diff
@@ -46,13 +48,11 @@ type GenesisAction struct {
 // Creator is the Creator of the genesis block
 type Creator struct {
 	PubKey string `yaml:"pubKey"`
-	PriKey string `yaml:"priKey"`
```

Comment:
Can we decouple removing prikey from this PR. IMHO, it's better to do this after we make the genesis to be creating the states then the actions completely.

Reply:
yes, let's do it in next release